### ISSUE TITLE
"GPS signal lost" message: Add option to disable in tunnels or disable entirely

### ISSUE THREAD

**Issue by cmeeren** [osmandapp/Osmand#2723]
When I'm driving through a tunnel, I don't need OsmAnd to tell me that the GPS signal has been lost. (Also I don't need OsmAnd to tell me that the signal has been restored after exiting the tunnel).

I suggest to add an option to the voice guidance settings:

    Disable alert on GPS Signal lost/restored
        - In tunnel
        - Always

The "In tunnel" option would mute the "GPS signal lost" if it occurs in or, say, some seconds after exiting a tunnel. Furthermore, if the GPS signal has been lost in a tunnel, the subsequent "GPS signal restored" message should also be muted.

The "Always" option should be self-explanatory.

Using OsmAnd 2.3.5.

**Comment by littlehelper** [comment]
What about an additional option to switch off these two voice guidance messages and just play less pushy "GPS signal lost" and "GPS signal restored" sounds instead? The residual voice guidance shall not be affected.

**Comment by jlamoure** [comment]
To resurrect a 3y.o. issue. My phone has GPS issues and it's terrible to have a car full of people listen to a voice announcement every minute or two of a long drive. I think this should be switch-off-able.

**Comment by breversa** [comment]
Alternatively, are you trying to solve your GPS issue ? Send me a message if you need help, so as not to clutter this thread. :-)

**Comment by jlamoure** [comment]
I'm working on that, all the same it would be nice to have a box to uncheck alongside the other items in the Announce options.

**Comment by vshcherb** [comment]
Yes, indeed we could add that setting to Announce options, will keep it in our backlog
